nixos/modules/services/networking/openvpn.nix

```diff
@@ -181,6 +191,12 @@ in
             '';
           };
 
+          authUserPassFile = mkOption {
+            default = null;
+            description = lib.mdDoc ''Specify a file path to use as `auth-user-pass ''${authUserPass}`. File will not be included in the nix store unless you specify it as a path `authUserFile = ./myauthfile`'';
```

Comment:
We don't use mdDoc anymore

Reply:
This only tells me what I shouldn't do, not what I should do :(

What's the replacement?

Reply:
> What's the replacement?

Just a literal string. lib.mdDoc just return the value you pass it now, so it's effectively a no-op. Documentation is now assumed to be markdown.